onnx2kerastl/convolution_layers.py

```diff
@@ -163,9 +157,15 @@ def convert_conv(node, params, layers, lambda_func, node_name, keras_name):
             conv_args['padding'] = 'same'
         else:
             conv_args['padding'] = 'valid'
-
         partial_conv = partial(keras.layers.Conv1D, **conv_args)
-        layers[node_name] = permute_wrap_conv_if_constant(partial_conv, input_0, is_constant, weights[0].shape[-2])
+        if has_bias:
+            res = permute_wrap_conv_if_constant(partial_conv, input_0, is_constant, weights[0].shape[-2])
+            res_shape = np.asarray(keras.backend.int_shape(res))
+            bias_dim = np.argwhere(res_shape == bias.shape)[0][0]
+            expanded_dims = [dim for dim in range(len(res_shape)) if dim != bias_dim]
+            layers[node_name] = res + np.expand_dims(bias, expanded_dims)
+        else:
+            layers[node_name] = permute_wrap_conv_if_constant(partial_conv, input_0, is_constant, weights[0].shape[-2])
```

Comment:
small styling suggestion:
```suggestion
        res = permute_wrap_conv_if_constant(partial_conv, input_0, is_constant, weights[0].shape[-2])
        if has_bias:
            res_shape = np.asarray(keras.backend.int_shape(res))
            bias_dim = np.argwhere(res_shape == bias.shape)[0][0]
            expanded_dims = [dim for dim in range(len(res_shape)) if dim != bias_dim]
            res = res + np.expand_dims(bias, expanded_dims)

        layers[node_name] = res
```